geopandas/geodataframe.py

```diff
@@ -970,10 +970,35 @@ def iterfeatures(self, na="null", show_bbox=False, drop_id=False):
 
                 yield feature
 
-    def _to_geo(self, **kwargs):
+    def to_geo_dict(self, **kwargs):
```

Comment:
Since it will be public API now, can we make the keyword args explicit? Rather than passing them via `kwargs` and pointing users to `iterfeatures`.

Reply:
Yes sure! I do that.

And by the way do you know what is the problem with the failing tests ?

Reply:
Unrelated, you can ignore those.